### ISSUE TITLE
Pressing `SPC f p` list file names with missing two first letters

### ISSUE THREAD

**Issue by brorbw** [hlissner/doom-emacs#3601]
**What did you expect to happen?**
The entire file name appeared and that opening one of the files would yield that file.

**What actually happened?**
Each file name is missing the first two letters and opens a new file with this wrong filename. 
![Image of error](https://i.imgur.com/dNsl46y.png)

**Additional details:**
The function called is `(doom/find-file-in-private-config)`
My dotfiles are symlinks. `.doom.d -> /Users/username/.dotfiles/doom.d`

**System information:**
<details>

```
emacs   version    27.0.91
				features   RSVG IMAGEMAGICK GLIB NOTIFY KQUEUE ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS PDUMPER LCMS2 GMP
				build      Jul 18, 2020
				buildopts  (--disable-dependency-tracking --disable-silent-rules --enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-plus@27/27.0.91/share/info/emacs --prefix=/usr/local/Cellar/emacs-plus@27/27.0.91 --with-xml2 --with-gnutls --without-dbus --with-imagemagick --with-modules --with-rsvg --with-ns --disable-ns-self-contained)
				windowsys  ns
				daemonp    server-running
doom    version    2.0.9
				build      HEAD -> develop 77f554c63 2020-07-17 23:34:44 -0400
				dir        ~/.dotfiles/doom.d/
system  type       darwin
				config     x86_64-apple-darwin19.2.0
				shell      /bin/zsh
				uname      Darwin 19.2.0 Darwin Kernel Version 19.2.0: Sat Nov  9 03:47:04 PST 2019; root:xnu-6153.61.1~20/RELEASE_X86_64 x86_64
				path       (/usr/local/bin ~/.emacs.d/bin ~/.node_modules/bin /usr/local/opt/coreutils/libexec/gnubin ~/.cargo/bin /usr/local/bin /usr/bin /bin /usr/sbin /sbin /Library/TeX/texbin /opt/X11/bin /usr/local/Cellar/node/10.11.0/bin /usr/local/sbin /Users/noname/Projects/depot_tools ~/.antigen/bundles/robbyrussell/oh-my-zsh/lib ~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/git ~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/heroku ~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/pip ~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/lein ~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/command-not-found ~/.antigen/bundles/zsh-users/zsh-syntax-highlighting ~/.antigen/bundles/sobolevn/wakatime-zsh-plugin ~/.go/bin /usr/local/opt/go/libexec/bin ~/Library/Python/3.8/bin /usr/local/Cellar/emacs-plus@27/27.0.91/libexec/emacs/27.0.91/x86_64-apple-darwin19.2.0/)
config  envfile    envvar-file
				elc-files  0
				modules    (:completion company (ivy +fuzzy +icons) :ui doom doom-dashboard doom-quit hl-todo modeline ophints (popup +defaults) pretty-code treemacs vc-gutter vi-tilde-fringe workspaces zen :editor (evil +everywhere) file-templates fold multiple-cursors snippets word-wrap :emacs dired electric undo vc :checkers syntax spell grammar :tools debugger docker editorconfig (eval +overlay) lookup (lsp +lsp-ivy) magit make pdf prodigy :lang (cc +lsp) clojure common-lisp data emacs-lisp (go +lsp) json javascript latex markdown org python rest scheme sh web yaml :app calendar :config (default +bindings +smartparens))
				packages   ((company-fuzzy))
				unpin      (n/a)
				elpa       (n/a)
```
</details>

**Comment by hlissner** [comment]
Duplicate of and fixed in #3586. Update Doom to fix the issue.

**Comment by brorbw** [comment]
Just updated and was about to close the ticket myself. Can confirm working
